__tests__/app.test.js

```diff
@@ -27,4 +27,18 @@ describe("CORE: GET - /api/topics", () => {
             });
           });
       });
+})
+
+describe("CORE: GET - /api", () => {
+  test("200: Endpoint should contain an object with all endpoints", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toHaveProperty("GET /api", expect.any(Object));
+            expect(body).toHaveProperty("GET /api/topics", expect.any(Object)); 
+            expect(body).toHaveProperty("GET /api/articles", expect.any(Object)); 
+
```

Comment:
This test works now but you will have to keep updating this test as you keep adding data to your endpoints.json. 

Try and see if you can make this dynamic - if you required the endpoints.json file in your controller maybe you could require it in for use here?